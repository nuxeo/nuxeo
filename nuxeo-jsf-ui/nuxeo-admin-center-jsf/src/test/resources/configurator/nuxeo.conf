# Configuration file for Nuxeo

JAVA_HOME="/usr/lib/jvm/java-6-sun"
JAVA_OPTS="-Xmx768m -XX:MaxPermSize=256m"

LOG_DIR="logs"
PID_DIR="bin"

# See templates/README.txt for customization parameters.
nuxeo.templates=default,custom
#nuxeo.bind.address=0.0.0.0

test.nuxeo.conf = true
### BEGIN - DO NOT EDIT BETWEEN BEGIN AND END ### 30/11/2011 00:00:00
#nuxeo.templates=default,custom
nuxeo.templates=default,oldchange
### END - DO NOT EDIT BETWEEN BEGIN AND END ###
