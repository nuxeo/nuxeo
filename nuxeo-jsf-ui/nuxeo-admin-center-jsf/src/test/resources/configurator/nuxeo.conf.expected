# Configuration file for Nuxeo

JAVA_HOME="/usr/lib/jvm/java-6-sun"
JAVA_OPTS="-Xmx768m -XX:MaxPermSize=256m"

LOG_DIR="logs"
PID_DIR="bin"

# See templates/README.txt for customization parameters.
nuxeo.templates=postgresql,oldchange
#nuxeo.bind.address=0.0.0.0

test.nuxeo.conf = true
### BEGIN - DO NOT EDIT BETWEEN BEGIN AND END ### JJ/MM/AAAA hh:mm:ss
#nuxeo.bind.address=0.0.0.0
nuxeo.bind.address=127.0.0.1
#server.status.key=
server.status.key=rAnd0MKey
#test.default.nuxeo.defaults=true
test.default.nuxeo.defaults=false
### END - DO NOT EDIT BETWEEN BEGIN AND END ###
